build.gradle

```diff
@@ -35,7 +35,7 @@ buildscript {
         js_resource_folder = "src/test/resources/job-scheduler"
         common_utils_version = System.getProperty("common_utils.version", opensearch_build)
         job_scheduler_version = System.getProperty("job_scheduler.version", opensearch_build)
-        bwcVersionShort = "2.19.0"
+        bwcVersionShort = "2.20.0"
```

Comment:
since 2.20 wont be released, should this stay 2.19 or it fails if its not 2.20?

Reply:
without doing it bwc test failed. OpenSearch current major (3.0) is wire compatible only with last minor of previous major (2.20). Read https://github.com/opensearch-project/OpenSearch/issues/5076

Reply:
I see, I assume that might need to change in future though if 2.20 is never released we will want for example 2.19.x be bwc with 3.0?